otherlibs/stdune/ansi_color.mli

```diff
@@ -3,6 +3,9 @@ module Style : sig
 
   val to_dyn : t -> Dyn.t
 
+  (** [to_string t] is the underlyingstring value of [t] *)
+  val to_string : t -> string
```

Comment:
In order to be able to match on the ansi color values I had to expose this @rgrinberg. I don't think it is harmful but it should perhaps have a better name.

Reply:
What about just turning this type into a proper variant? Matching on magic numbers isn't the way to go.